kuma/wiki/models.py

```diff
@@ -1663,6 +1613,7 @@ class Revision(models.Model):
 
     summary = models.TextField()  # wiki markup
     content = models.TextField()  # wiki markup
+    tidied_content = models.TextField(blank=True)  # wiki markup tidied up
```

Comment:
Ah, this is a good way to preserve the submitted data (in the `content` field) **and** store the tidied content. But should we populate the `tidied_content` field when it's accessed via `get_tidied_content()` or should we just populate it when the `Revision` is saved?